neurodsp/plts/time_series.py

```diff
@@ -48,9 +50,15 @@ def plot_time_series(times, sigs, labels=None, colors=None, ax=None, **kwargs):
 
     ax = check_ax(ax, (15, 3))
 
-    times = repeat(times) if (isinstance(times, np.ndarray) and times.ndim == 1) else times
     sigs = [sigs] if (isinstance(sigs, np.ndarray) and sigs.ndim == 1) else sigs
 
+    xlabel = 'Time (s)'
+    if times is None:
+        times = create_samples(len(sigs[0]))
+        xlabel = 'Samples'
```

Comment:
`xlabel` should default to `Time (s)` somewhere. When I plot, specifying a time array, there is no xlabel. Maybe:
```
xlabel = 'Samples' if times is None else 'Time (s)'
times = create_samples(len(sigs[0])) if times is None else times
```
or 
```
if times is None:
    times = create_samples(len(sigs[0]))
    xlabel = 'Samples'
else:
    xlabel = 'Time (s)'
```

Reply:
Hmmm...., that's weird that the times label isn't being added - I'm pretty sure that was working when I tested this.

`xlabel` is defined in line 55, which should provide it with a default value that gets used if the if statement in line 56 is not invoked. This is (I think) effectively equivalent to the suggestions you mentioned, so I'm not sure why it wouldn't get applied?

Reply:
Whoops, sorry! I don't know how I didn't see line 55. The plot  I was using had additional custom plt calls that was messing with the xlabel.